thunder/tests/test_grad.py

```diff
@@ -35,6 +35,7 @@
     # Finite difference approximation doesn't work for this function
     "embedding",
     "index_put",
+    "batch_norm",
```

Comment:
How do automatic tests fail?

Reply:
it fails with somewhere requires `TensorProxy` but gets `torch.Tensor`, I think it's related to the `_make_differentiable_wrapper` function, when we have multiple inputs and some require grad and some don't

Reply:
I'll follow up to debug why that happens